content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -499,6 +499,18 @@ The number of datagrams dropped because of an error while writing to Datadog.
 : **Metric type**: count<br>
 The number of metrics dropped because the internal receiving channel is full (only when using `WithChannelMode()`). Starting with version `3.6.0` of the Go client.
 
+`datadog.dogstatsd.client.aggregated_context`
+: **Metric type**: count<br>
+The number of contexts aggregated when client side aggregation is enabled. Starting with version `4.3.0`.
+
+`datadog.dogstatsd.client.aggregated_context_by_type`
+: **Metric type**: count<br>
+The number of contexts aggregated by type when client side aggregation is enabled. Starting with version `4.3.0`. The metric is enabled by default starting `5.0.0` but requires `WithDevMode` for the versions before or equal to `4.8.1`. The metric is tagged with either `metrics_type:gauge`, `metrics_type:count`, `metrics_type:set`, `metrics_type:timing`, `metrics_type:histogram` nor `metrics_type:distribution`.
```

Comment:
We might release a new bug fix on v4 someday so I would just say:

```suggestion
The number of contexts aggregated by type when client side aggregation is enabled. Starting with version `4.3.0`. The metric is enabled by default starting `v5.0.0` but requires `WithDevMode()` option for the previous versions.
```

Also are we planning on blocking this commit until the release of both Java and Go new major ?

Reply:
Also, let's be consistent and use `vX.Y.Z` or `X.Y.Z` everywhere, we currently have a bit of a mix. I think I like the `v` prepended because that's how we tag the repositories.

Reply:
Outdated as https://github.com/DataDog/documentation/pull/11900/files describes the same metrics. Changed removed.